src/m365/spfx/commands/project/project-upgrade/upgrade-1.0.2.ts

```diff
@@ -4,8 +4,8 @@ import { FN008003_CFG_TSL_preferConst } from "./rules/FN008003_CFG_TSL_preferCon
 import { FN010001_YORC_version } from "./rules/FN010001_YORC_version.js";
 
 export default [
-  new FN001019_DEP_knockout('3.4.0'),
-  new FN001020_DEP_types_knockout('3.4.39'),
+  new FN001019_DEP_knockout({ packageVersion: '3.4.0' }),
+  new FN001020_DEP_types_knockout({ packageVersion: '3.4.39' }),
   new FN008003_CFG_TSL_preferConst(),
   new FN010001_YORC_version('1.0.2')
```

Comment:
It seems like this rule still uses separate args instead of options. Update. Also, check if there are other rules that use separate args and refactor them to use an object instead.